Public/Start-AtomicGUI.ps1

```diff
@@ -0,0 +1,268 @@
+function Start-AtomicGUI {
+    param (
+        [Int] $port = 8487
+    )
+    # Install-Module UniversalDashboard if not already installed
+    $UDcommunityInstalled = Get-InstalledModule -Name "UniversalDashboard.Community" -ErrorAction:SilentlyContinue
+    $UDinstalled = Get-InstalledModule -Name "UniversalDashboard" -ErrorAction:SilentlyContinue
+    if (-not $UDcommunityInstalled -and -not $UDinstalled) { 
+        Write-Host "Installing UniversalDashboard.Community"
+        Install-Module -Name UniversalDashboard.Community -Scope CurrentUser -Force
```

Comment:
I think auto-installing the dependency is fine for now but we'll need to be more consistent about how we communicate/handle external module dependencies in the future.